nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -113,9 +114,10 @@
 import static org.apache.commons.lang3.StringUtils.trimToEmpty;
 
 @SupportsBatching
-@Tags({"http", "https", "rest", "client"})
+@Tags({"http", "https", "rest", "client","ntlm"})
 @InputRequirement(Requirement.INPUT_ALLOWED)
 @CapabilityDescription("An HTTP client processor which can interact with a configurable HTTP Endpoint. The destination URL and HTTP Method are configurable."
+    + " Also supports NTLM authentication to communicate with a sharepoint webservice"
     + " FlowFile attributes are converted to HTTP headers and the FlowFile contents are included as the body of the request (if the HTTP Method is PUT, POST or PATCH).")
```

Comment:
Unless NTLM _only_ works with sharepoint, there isn't a need to specify it here.  If you _do_ specify it, you might want to capitalize it correctly.

Reply:
Adjusted the Description, works with NTLM in general